src/components/NewEntry.js

```diff
@@ -0,0 +1,230 @@
+import styled from 'styled-components/macro'
+import { nanoid } from 'nanoid'
+
+const colorArray = [
+  'weiß',
+  'schwarz',
+  'grün',
+  'gelb',
+  'blau',
+  'rosa',
+  'lila',
+  'pink',
+  'keine',
+]
+
+const patternArray = [
+  'einfarbig',
+  'geblümt',
+  'gemustert',
+  'gepunktet',
+  'gestreift',
+  'kariert',
+  'leopardenmuster',
+  'nadelstreifen',
+  'schlangenmuster',
+  'keine',
+]
+
+const materialArray = [
+  'Baumwolle',
+  'Denim',
+  'Leinen',
+  'Mesh',
+  'Satin',
+  'Seide',
+  'Spitze',
+  'Wolle',
+]
+
+const fittingArray = [
+  'Figurbetont',
+  'Loose Fit',
+  'Oversize',
+  'Regular Fit',
+  'Relaxed Fit',
+]
+
+function NewEntry({ onNavigate, onNewEntry }) {
+  function handleSubmit(event) {
+    event.preventDefault()
+    const form = event.target
+
+    const { category, title, color, pattern, material, fitting } = form.elements
+
+    const newEntry = {
+      id: nanoid(),
+      category: category.value,
+      title: title.value,
+      color: color.value,
+      pattern: pattern.value,
+      material: material.value,
+      fitting: fitting.value,
+      imageUrl: '/images/bluse-floral.png', // cloudinary einbinden
```

Comment:
Comment is a reminder or unnecessary code?

Reply:
And I would add that you should write your comments in English.

Reply:
Actually comment as a reminder. Have it removed anyway ;)